Packs/CommonScripts/Scripts/ShowIncidentIndicators/ShowIncidentIndicators.py

```diff
@@ -32,7 +32,7 @@ def get_indicators_from_incident():
     """
     incident_id = demisto.incident()['id']
     indicators_query = {
-        "incident": incident_id
+        "investigationIDs": incident_id
```

Comment:
https://panw-global.slack.com/archives/C011BDBV3UP/p1676973323987969?thread_ts=1676913079.424499&cid=C011BDBV3UP

Reply:
I don't have the capacity for a test playbook at the moment, this is a quick bugfix for something seen on XSOAR 8

Reply:
Also this is a field display script so I don't think there's a good way to test it in a test playbook, however I tested it both on 6.10 and on 8.2